cl/opinion_page/views.py

```diff
@@ -39,6 +39,7 @@
 from cl.lib.ratelimiter import ratelimit_if_not_whitelisted
 from cl.lib.search_utils import (
     get_citing_clusters_with_cache,
+    get_parenthetical_groups_with_cache,
```

Comment:
Hm, I'm not sure the right place for this, but `search_utils` usually has something to do with searching. Maybe we rename `filter_parentheticals.py` to be `parenthetical_utils` or something, and put this in there?

Reply:
`search_utils` does seem to have a number of methods not related to search (citing opinions, related clusters, etc.). We can make this change, but this method is arguably much more conceptually similar to the methods around it in this file -- it computes, with caching, some related piece of information about a given opinion that were going to display -- than the parenthetical functionality.

Maybe we should move a bunch of the opinion page-related helpers in search_utils into a new utils file?

Reply:
Go figure. We can get back to this some other time. Doesn't have to be here.